src/main/plugins/workflow-plugin/workflow-plugin.ts

```diff
@@ -8,21 +8,26 @@ import { WorkflowExecutionStep } from "./workflow-execution-argument";
 import { WorkflowExecutionArgumentType } from "./workflow-execution-argument-type";
 import { Workflow } from "./workflow";
 import { defaultWorkflowIcon } from "../../../common/icon/default-icons";
+import { WindowsShell, MacOsShell } from "../commandline-plugin/shells";
+import { CommandlineOptions } from "../../../common/config/commandline-options";
 
 export class WorkflowPlugin implements SearchPlugin {
     public pluginType = PluginType.Workflow;
     private config: WorkflowOptions;
+    private commandConfig: CommandlineOptions;
```

Comment:
Can we rename this to `commandlineOptions`? I think it would be a more accurate name.